Fiction notes/GDL_PorosityDistribution_collector.md

```diff
@@ -64,4 +72,10 @@ Section1 |  Section2 | Description |
 结论 | 
 结论1 | -- | cell performance and optimum porosity distribution are similar for the continuous or discrete porosity distributions and that a discrete variation of porosity may be used in practice just as effectively
 结论2 | -- | for E_{cell}=0.50V the local current density decreases monotonically along the channel since the variation of the porosity is smaller and the effect of the decreasing species concentration dominates the local current density distribution along the entire channel
-结论3 | -- | the optimum porosity was found to decrease with increase in temperature, and increase with increase with cell voltage, uniformity criterion, cathode mass flow rate and cathode relative humidity, whereas a non-monotonic relationship was observed with the changes in the operating pressure
\ No newline at end of file
+结论3 | -- | the optimum porosity was found to decrease with increase in temperature, and increase with increase with cell voltage, uniformity criterion, cathode mass flow rate and cathode relative humidity, whereas a non-monotonic relationship was observed with the changes in the operating pressure
+
+## Fragments <a name="fragments"></a>
```

Comment:
add the first idea from a single article, the format didn't include the article itself